website/docs/contract-test-collection.md

```diff
@@ -459,130 +1061,347 @@ authors: Taquito
+    });
 ```
+</TabItem>
+  <TabItem value="metadata">
+
+```js
 name: Taquito test with valid metadata
 description: This is metadata test for Taquito integration tests with the Ligo Taco shop contract modified to include metadata in storage
 version: 7.1.0-beta.0
 license: MIT
 homepage: https://github.com/ecadlabs/taquito
 ```
+</TabItem>
+</Tabs>
 
 ## Tzip16IPFSContract
 
 [KT1NP2ZVLxWaSBQryDzUujmdv27ubJWZRckv](https://better-call.dev/hangzhou2net/KT1NP2ZVLxWaSBQryDzUujmdv27ubJWZRckv/code)
 
 #### Entrypoints:
-  * default
-#### Storage:
+
+- default
+
+<Tabs
+defaultValue="michelson"
+values={[
+{label: 'Michelson Storage', value: 'michelson'},
+{label: 'Taquito Storage', value: 'taquito'},
+{label: 'Metadata', value: 'metadata'}
+]}>
+<TabItem value="michelson">
+
+
 ```js
 storage (pair (big_map %metadata string bytes)
               (map %taco_shop_storage nat (pair (nat %current_stock) (mutez %max_price))));
 ```
-#### Metadata:
+
+</TabItem>
+  <TabItem value="taquito">
+
+```js
+const uri = 'ipfs://QmXnASUptTDnfhmcoznFqz3S1Mxu7X1zqo2YwbTN3nW52V';
+    const bytesUrl = char2Bytes(uri);
+
+    const metadataBigMap = new MichelsonMap();
+    metadataBigMap.set('', bytesUrl);
+
+    const tacoShopStorageMap = new MichelsonMap();
+
+    const op = await tezos.contract.originate({
+      code: tacoContractTzip16,
+      storage: {
+        metadata: metadataBigMap,
+        taco_shop_storage: tacoShopStorageMap,
+      },
+    });
 ```
+</TabItem>
+  <TabItem value="metadata">
+
+```js
 name: Taquito test with valid metadata
 description: This is metadata test for Taquito integration tests with the Ligo Taco shop contract modified to include metadata in storage
 version: 7.1.0-beta.0
 license: MIT
 homepage: https://github.com/ecadlabs/taquitoj
 ```
+</TabItem>
+</Tabs>
 
-## Tzip16OffChainContractOne
+## Tzip16OffChainContractJSON
 
 [KT1Vms3NQK8rCQJ6JkimLFtAC9NhpAq9vLqE](https://better-call.dev/hangzhou2net/KT1Vms3NQK8rCQJ6JkimLFtAC9NhpAq9vLqE/code)
 
 #### Entrypoints:
-  * default
-#### Storage:
+
+- default
+
+<Tabs
+defaultValue="michelson"
+values={[
+{label: 'Michelson Storage', value: 'michelson'},
+{label: 'Taquito Storage', value: 'taquito'},
+{label: 'Metadata', value: 'metadata'}
+]}>
+<TabItem value="michelson">
+
 ```js
 storage (pair nat (big_map %metadata string bytes));
 ```
-#### Metadata:
+
+</TabItem>
+  <TabItem value="taquito">
+
+```js
+const metadataBigMAp = new MichelsonMap();
+    metadataBigMAp.set('', char2Bytes('tezos-storage:here'));
+    metadataBigMAp.set('here', char2Bytes(JSON.stringify(metadataViewsExample1)));
+
+    const op = await tezos.contract.originate({
+      code: contractCode,
+      storage: {
+        0: 7,
+        metadata: metadataBigMAp,
+      },
+    });
 ```
+</TabItem>
+  <TabItem value="metadata">
+
+```js
 description: This contract has bytes-returning off-chain-views.
 license: MIT
 ```
+</TabItem>
+</Tabs>
 
-## Tzip16OffChainContractTwo
+## Tzip16OffChainContractMultiply
```

Comment:
The name is not updated in the menu above and in the table below.

Reply:
fixed it.